### ISSUE TITLE
Create Full Interoperability Suite

### ISSUE THREAD

**Issue by lindluni** [hyperledger/fabric-test#263]
Fabric has three SDK's: Go, Node.js/Typescript, and Java, in order to maintain and verify functionality we should create a full-featured interoperability definition. This definition should define all of the functionality we will test, and then we should implement the Interoperability suite by replicating these test definitions in each SDK. This will allow us to verify that functionality remains valid and consistent.

**Comment by davidkel** [comment]
I don't think this should be done for the current SDKs (Go, Node, Java) as although Node and Java are almost functionally equivalent, Go definitely isn't. What is functionally equivalent between the 3 languages, there are probably subtle differences in the implementation and behaviour which could make this quite difficult to do.
However the new thin sdks have been created with this in mind and I believe is already part of the scenario tests in the gateway. What makes sense is to add the gateway scenario tests into the Interop suite
